docs/CONTRIBUTING.md

```diff
@@ -93,3 +93,12 @@ python setup.py typecheck
 # Code of Conduct
 
 [ONNX Open Source Code of Conduct](http://onnx.ai/codeofconduct.html)
+
+# Instructions for handling Corporate Contributor License Agreements (CCLAs)
+  * Step 1: Download the CCLA from [here]() 
+  * Step 2: Complete the form
+  * Step 3: Open a PR in the https://github.com/onnx/steering-committee/tree/master/CCLAs repo with the completed CCLA attached in the [CCLA directory](https://github.com/onnx/onnx/tree/master/community/CCLAs)
+  * Step 4: The steering committee members will be auto tagged for review including: @jspisak, @prasanthpul, @ciphr, @harryskim and Dilip Sequeira
```

Comment:
Let me setup a CODEOWNERS file in steering-committee repo so this step will not be needed. Any PR committing a file into the CCLA directory will tag the steering committee.

This will eliminate an extra manual step and also reduce the number of places we have to keep steering committee membership updated